config/config.json

```diff
@@ -17,10 +17,10 @@
     "arbitrum_sepolia": {
       "rlcCrossChainTokenAddress": "0x0000000000000000000000000000000000000000",
       "rlcCrossChainTokenCreatexSalt": "0x0000000000000000000000000000000000000000000000000000000000000000",
-      "approvalRequired": false,
       "iexecLayerZeroBridgeAddress": "0x0000000000000000000000000000000000000000",
       "iexecLayerZeroBridgeCreatexSalt": "0x0000000000000000000000000000000000000000000000000000000000000000",
-      "lzEndpointAddress": "0x6EDCE65403992e310A62460808c4b910D972f10f", 
+      "lzEndpointAddress": "0x6EDCE65403992e310A62460808c4b910D972f10f",
+      "approvalRequired": false,
```

Comment:
Change it for sepolia also to be consistent

Reply:
I mean to have the same order of params in both chains Arbitrum Sepolia and Seplia.
Ne worries, we'll clean it